CHANGELOG.md

```diff
@@ -215,11 +215,40 @@ All notable changes to this project will be documented in this file.
 - Deprecated `GET /agents/{agent_id}/group/is_sync` endpoint. ([#12365](https://github.com/wazuh/wazuh/issues/12365))
 - Deprecated `GET /manager/stats/analysisd`, `GET /manager/stats/remoted`, `GET /cluster/{node_id}stats/analysisd`, and `GET /cluster/{node_id}stats/remoted` API endpoints. ([#14230](https://github.com/wazuh/wazuh/pull/14230))
 
-### Ruleset
+### Ruleset 
+
+#### Added
+
+- Adding support for new sysmon events. ([#13594](https://github.com/wazuh/wazuh/pull/13594))
+- Adding new detection rules using Sysmon ID 1 events. ([#13595](https://github.com/wazuh/wazuh/pull/13595))
+- Adding new detection rules using Sysmon ID 3 events. ([#13596](https://github.com/wazuh/wazuh/pull/13596))
+- Adding new detection rules using Sysmon ID 7 events. ([#13630](https://github.com/wazuh/wazuh/pull/13630))
+- Adding new detection rules using Sysmon ID 8 events. ([#13637](https://github.com/wazuh/wazuh/pull/13637))
+- Adding new detection rules using Sysmon ID 10 events. ([#13639](https://github.com/wazuh/wazuh/pull/13639))
+- Adding new detection rules using Sysmon ID 11 events. ([#13631](https://github.com/wazuh/wazuh/pull/13631))
+- Adding new detection rules using Sysmon ID 13 events. ([#13636](https://github.com/wazuh/wazuh/pull/13636))
+- Adding rules for Sysmon ID 20 events. ([#13673](https://github.com/wazuh/wazuh/pull/13673))
```

Comment:
Why is this line using a different format with respect to all previous ones?

Reply:
it is not a different format, it is a new file that wasn't present before. It is not detection related change.

Reply:
Anyway, I did change it